it/v1/endpoints/CreateAndAmendEmploymentExpensesControllerISpec.scala

```diff
@@ -294,9 +178,109 @@ class CreateAndAmendEmploymentExpensesControllerISpec extends IntegrationBaseSpe
           (SERVICE_UNAVAILABLE, "SERVICE_UNAVAILABLE", INTERNAL_SERVER_ERROR, StandardDownstreamError)
         )
 
-        input.foreach(args => (serviceErrorTest _).tupled(args))
+        val extraTysErrors = Seq(
+          (BAD_REQUEST, "INVALID_CORRELATION_ID", INTERNAL_SERVER_ERROR, StandardDownstreamError),
+          (UNPROCESSABLE_ENTITY, "TAX_YEAR_NOT_SUPPORTED", BAD_REQUEST, RuleTaxYearNotSupportedError)
+        )
+
+        (errors ++ extraTysErrors).foreach(args => (serviceErrorTest _).tupled(args))
       }
     }
   }
 
+  private trait Test {
+
+    val nino: String = "AA123456A"
+
+    def taxYear: String
+
+    val amount: BigDecimal = 123.12
+
+    def requestBodyJson: JsValue = Json.parse(
+      s"""
+         |{
+         |    "expenses": {
+         |        "businessTravelCosts": $amount,
+         |        "jobExpenses": $amount,
+         |        "flatRateJobExpenses": $amount,
+         |        "professionalSubscriptions": $amount,
+         |        "hotelAndMealExpenses": $amount,
+         |        "otherAndCapitalAllowances": $amount,
+         |        "vehicleExpenses": $amount,
+         |        "mileageAllowanceRelief": $amount
+         |    }
+         |}
+         |""".stripMargin
+    )
+
+    val hateoasResponse: JsValue = Json.parse(s"""
+                                                 |{
```

Comment:
Can this block be moved to the left somewhat? Maybe line it up with the one above.